app/models/user.rb

```diff
@@ -107,6 +107,11 @@ def inactive_message
     "Sorry, this account has been disabled."
   end
 
+  # Returns all users who match the query.
+  def self.search_from_query(members, query)
+    User.enabled.where.not(id: members).where(arel_table[:username].matches(query)).pluck(:username)
```

Comment:
maybe we could further improve on this. Extract scopes, refactor to scope.

1.
```
def self.search_from_query(members, query)
  User.enabled.where.not(id: members).where(arel_table[:username].matches(query))
end
```

2.
```
def self.search_from_query(members, query)
  enabled.where.not(id: members).where(arel_table[:username].matches(query))
end
```

3.
```
scope :search_from_query, -> (members, query) {
  enabled.where.not(id: members).where(arel_table[:username].matches(query))
}
```

4.
```
scope :name_matches, ->(query) { where(arel_table[:username].matches(query)) }
scope :search_from_query, -> (members, query) { enabled.where.not(id: members).name_matches(query) }
```